doc/manual/source/store/resolution.md

```diff
@@ -0,0 +1,61 @@
+# Derivation Resolution
+
+To *resolve* a derivation is to replace its [inputs] with the simplest inputs --- plain store paths --- that denote the same store objects.
+
+Derivations that only have store paths as inputs are likewise called *resolved derivations*.
+(They are called that whether they are in fact the output of derivation resolution, or just made that way without non-store-path inputs to begin with.)
+
+[Deriving paths][deriving-path] intentionally make it possible to refer to the same [store object] in multiple ways.
+This is a consequence of content-addressing, since different derivations can produce the same outputs, and the same date can also be manually added to the store.
+This is also a consequence even of input-addressing, as an output can be referred to by derivation and output name, or directly by its store path input address.
```

Comment:
The fact that an input-addressed derivation output is an output with an "input address" is distracting because of the whole in versus out business.

```suggestion
This is also a consequence even of input-addressing, as an output can be referred to by derivation and output name, or directly by its [computed](./outputs/input-address.md) store path.
```